Makefile

```diff
@@ -29,7 +29,7 @@ CROSS_BUILD_BINDIR := $(OUTPUT_DIR)/bin
 GIT_VERSION        := $(shell git describe --dirty --tags --always --match='v*')
 GIT_SHA            := $(shell git rev-parse HEAD)
 GIT_BRANCH         := $(shell git rev-parse --abbrev-ref HEAD)
-VERSION            ?= $(GIT_VERSION)
+VERSION            := $(GIT_VERSION)
```

Comment:
This removes the ability to set version externally, doesn't?

Reply:
command-line assignments have the highest precedence and will override any assignment made within the Makefile itself.